hw/formal/formal_conn.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Script to run connectivity test using Jasper Gold
+#
+# Usage: To run connectivity test on system level modules, type
+#   formal_conn -f top_earlgrey.csv
+#
+# More options:
+# -top: which top_level module to run
+# -p:  provide core file path
+# -gui: run with GUI
+# -cov: run coverage
+# -f: path to the csv file
+# -t: choose which formal tool to use, current only support jaspergold
+#
+# Example:
+#   formal_conn -gui -cov -f top_earlgrey.csv
+#
+
+export TOP="top_earlgrey"
```

Comment:
I'd suggest adding `set -e` here: the behaviour on an unexpected error isn't great, but it's better than stumbling on to later commands that can get really confused.

Reply:
Thank you Rupert for pointing that out. I was not aware that bash will ignore errors in scripts if we do not set this one..

Reply:
IIUC this file will go away later, correct, now that DVSim support has been added?

Reply:
Yes yes, this is just a temp solution to use for GUI mode, it will go away once dvsim support it :)